src/query/api/v1/handler/placement/add.go

```diff
@@ -36,13 +38,26 @@ import (
 )
 
 const (
-	// AddURL is the url for the placement add handler (with the POST method).
-	AddURL = handler.RoutePrefixV1 + "/placement"
+	// OldM3DBAddURL is the old url for the placement add handler, maintained for
+	// backwards compatibility.
+	OldM3DBAddURL = handler.RoutePrefixV1 + "/placement"
+
+	// M3DBAddURL is the url for the placement add handler (with the POST method)
+	// for the M3DB service.
+	M3DBAddURL = handler.RoutePrefixV1 + "/services/m3db/placement"
```

Comment:
Hm, are we worried about breaking all our guides/etc?  I guess it's ok, at some point we'll probably have to version our API docs similar to ElasticSearch, etc so that we can go and see what the APIs were at a specific version (for users still running an older version).

Reply:
Is it better to write: 
`M3DBAddURL = handler.RoutePrefixV1 + "/services/" + M3DBServiceName + "/placement"`
in case you update the service in one place and forgot the others. 

Same goes to the m3agg url as well. Also nit: use `m3aggregator` instead of `m3agg` everywhere?

Reply:
Also consider making a const for `"/services/" + M3DBServiceName + "/placement"`, I see that string being reused in a lot of places.

Reply:
Agreed can we break up the consts?